nservicebus/azure-storage-persistence/configuration_code_azure_[6,].partial.md

```diff
@@ -0,0 +1,13 @@
+### Configuration with Code
```

Comment:
This part is identical for asp 1 version, version switching will handle snippets versioning, we might do without this section then. Optionally we could have only partials for "For Sagas, Subscriptions and for Timeouts:" part.

Or am I missing something @Scooletz ?